azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDaoImpl.java

```diff
@@ -36,14 +58,15 @@ public ImageTypeDaoImpl(DatabaseOperator databaseOperator) {
   @Override
   public int createImageType(ImageType imageType) {
     final SQLTransaction<Integer> insertAndGetSpaceId = transOperator -> {
-      String currentTime = DateTime.now().toLocalDateTime().toString();
-      transOperator.update(INSERT_IMAGE_TYPE, imageType.getType(), imageType.getDescription(),
-          true, imageType.getDeployable().getName(), currentTime, imageType.getCreatedBy());
+      // insert image type record
+      transOperator.update(INSERT_IMAGE_TYPE, imageType.getName(), imageType.getDescription(),
+          true, imageType.getDeployable().getName(), imageType.getCreatedBy(), imageType.getCreatedBy());
```

Comment:
Can you please use getModifiedBy over here as well? It will reduce the confusion?

Reply:
Done.